invenio_vocabularies/contrib/subjects/config.py

```diff
@@ -2,30 +2,36 @@
 #
 # Copyright (C) 2021 CERN.
 # Copyright (C) 2021 Northwestern University.
+# Copyright (C) 2024 University of Münster.
 #
 # Invenio-Vocabularies is free software; you can redistribute it and/or
 # modify it under the terms of the MIT License; see LICENSE file for more
 # details.
 
 """Subjects configuration."""
 
+from flask import current_app
 from invenio_i18n import lazy_gettext as _
 from invenio_records_resources.services import SearchOptions
 from invenio_records_resources.services.records.components import DataComponent
+from werkzeug.local import LocalProxy
 
 from ...services.components import PIDComponent
 from ...services.querystr import FilteredSuggestQueryParser
 
+subject_schemes = LocalProxy(
+    lambda: current_app.config["VOCABULARIES_SUBJECTS_SCHEMES"]
+)
+
 
 class SubjectsSearchOptions(SearchOptions):
     """Search options."""
 
     suggest_parser_cls = FilteredSuggestQueryParser.factory(
         filter_field="scheme",
         fields=[  # suggest fields
-            "subject^100",
-            "subject._2gram",
-            "subject._3gram",
+            "title.*^100",
```

Comment:
```suggestion
            "subject",
            "title.*^100",
```

- We can add back the `subject` field
- Regarding the localized titles, see issue and comment at https://github.com/inveniosoftware/invenio-records-resources/pull/594#issuecomment-2288980884 on how this should be handled.